frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "gulp-protractor": "0.0.11",
     "gulp-replace": "^0.5.3",
     "gulp-ruby-sass": "^0.7.1",
+    "gulp-util": "^3.0.4",
```

Comment:
@hirtie-maxim I was seeing that too. If it's not too hard, can you split the commit and add an appropriate message to that effect?